Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM golang:1.15-alpine AS builder
+FROM golang:1.16-alpine AS builder
 WORKDIR /root
 
 RUN apk --no-cache add gcc musl-dev git
-COPY . /root 
+COPY . /root
 
 ENV APP_VERSION=v2.0.0
 RUN ls -h && go mod vendor \
     && CGO_ENABLED=0 GOARCH=amd64 go build \
     -o app cmd/aws-s3-proxy/main.go
 
-FROM alpine:3
+FROM scratch
```

Comment:
Won't you need a copy of `ca-certificates` so you can verify TLS when talking to a secure backend?